gradio/routes.py

```diff
@@ -73,7 +73,13 @@ def _render_str(content: Any) -> str:
 
 
 def toorjson(value):
-    return htmlsafe_json_dumps(value, dumps=ORJSONResponse._render_str)
+    return markupsafe.Markup(
```

Comment:
Is this the implementation of html_safe_json_dumps?

Reply:
Yup basically